### ISSUE TITLE
Level Zero API PDF

### ISSUE THREAD

**Issue by doppioandante** [oneapi-src/oneAPI-spec#275]
Hello, this is not probably the right tracker but I'm not sure where else to ask this.

The current Level Zero spec lives at https://spec.oneapi.com/level-zero/latest/index.html
For offline browsing a PDF version would be very useful.
Readthedocs allows [offline documentation generation](https://docs.readthedocs.io/en/latest/downloadable-documentation.html) but it needs to be enabled.
Would you be willing or able to enable that for the level zero API?

**Comment by rscohn2** [comment]
@zackwaters: Are you able to make a PDF?

**Comment by rscohn2** [comment]
@zackwaters Did you want to add a PDF?

**Comment by doppioandante** [comment]
Ping? I still feel this would be useful, and also easy enough to enable in the readthedocs site.

**Comment by zackwaters** [comment]
@doppioandante Yes, we do want to add a PDF for Level Zero and are working to generate it now. Will update soon.
